src/db_options.rs

```diff
@@ -3578,6 +3582,27 @@ pub enum ChecksumType {
     XXH3 = 4, // Supported since RocksDB 6.27
 }
 
+/// Used by [`PlainTableFactoryOptions`]
+#[repr(u8)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Default)]
```

Comment:
The rust version should be bumped. 1.60.0 doesn't support the `Default` trait for enums.

Reply:
That, or I could hand-impl the trait so we don't need an MSRV bump